pillar/heroku/micromasters.sls

```diff
@@ -0,0 +1,212 @@
+    MICROMASTERS_EMAIL_PORT: 587
+    MICROMASTERS_EMAIL_TLS: True
+    MICROMASTERS_EMAIL_USER: __vault__::secret-operations/mit-smtp>data>relay_username
+    MICROMASTERS_ENVIRONMENT: {{ env_data.env_name }}
+    MICROMASTERS_FROM_EMAIL: 'MITx MicroMasters <micromasters-support@mit.edu>'
+    MICROMASTERS_LOG_LEVEL: {{ env_data.MICROMASTERS_LOG_LEVEL }}
+    MICROMASTERS_SUPPORT_EMAIL: 'micromasters-support@mit.edu'
+    MICROMASTERS_USE_S3: True
+    NODE_MODULES_CACHE: False
+    OPEN_DISCUSSIONS_API_USERNAME: {{ env_data.OPEN_DISCUSSIONS_API_USERNAME }}
+    OPEN_DISCUSSIONS_BASE_URL: {{ env_data.OPEN_DISCUSSIONS_BASE_URL }}
+    OPEN_DISCUSSIONS_COOKIE_DOMAIN: {{ env_data.OPEN_DISCUSSIONS_COOKIE_DOMAIN }}
+    OPEN_DISCUSSIONS_COOKIE_NAME: {{ env_data.OPEN_DISCUSSIONS_COOKIE_NAME }}
+    OPEN_DISCUSSIONS_JWT_SECRET: __vault__::secret-{{ business_unit }}/open-discussions>data>jwt_secret
+    OPEN_DISCUSSIONS_REDIRECT_COMPLETE_URL: '/complete/micromasters'
+    OPEN_DISCUSSIONS_REDIRECT_URL: {{ env_data.OPEN_DISCUSSIONS_BASE_URL }}
+    OPEN_DISCUSSIONS_SITE_KEY: 'micromasters'
+    OPEN_EXCHANGE_RATES_APP_ID: __vault__::secret-{{ business_unit }}/open-exchange-rates>data>app_id
+    OPEN_EXCHANGE_RATES_URL: 'https://openexchangerates.org/api/'
+    OPENSEARCH_HTTP_AUTH: __vault__::secret-{{ business_unit }}/opensearch>data>http_auth
+    OPENSEARCH_INDEX: 'micromasters'
+    OPENSEARCH_URL: __vault__::secret-{{ business_unit }}/opensearch>data>url
+    PGBOUNCER_DEFAULT_POOL_SIZE: {{ env_data.PGBOUNCER_DEFAULT_POOL_SIZE }}
+    PGBOUNCER_MAX_CLIENT_CONN: {{ env_data.PGBOUNCER_MAX_CLIENT_CONN }}
+    SECRET_KEY: __vault__:gen_if_missing:secret-{{ business_unit }}/django>data>secret_key    
+    SENTRY_AUTH_TOKEN: __vault__::secret-{{ business_unit }}/sentry>data>auth_token                 
+    SENTRY_DSN: __vault__::secret-{{ business_unit }}/sentry>data>dsn
+    SENTRY_ORG_NAME: 'mit-office-of-digital-learning'
+    SENTRY_PROJECT_NAME: 'micromasters'
+    STATUS_TOKEN:  __vault__:gen_if_missing:secret-{{ business_unit }}/django>data>status_token                     
+    UWSGI_PROCESS_COUNT: 4
+    UWSGI_SOCKET_TIMEOUT: 1                      
+    UWSGI_THREAD_COUNT: 50
+    {% if env_data.env_name == 'production' %}
+    {% set pg_creds = salt.vault.cached_read('postgres-micromasters/creds/app', cache_prefix='heroku-micromasters') %}
+    ADWORDS_CONVERSION_ID: 935224753
+    CLIENT_ELASTICSEARCH_URL: '/api/v0/search/'
+    DATABASE_URL: postgres://{{ pg_creds.data.username }}:{{ pg_creds.data.password }}@{{ rds_endpoint }}/micromasters
+    FEATURE_PEARSON_EXAMS_SYNC: True
+    {% endif %}
+    {% if env_data.env_name != 'ci' %}
+    CLOUDFRONT_DIST: {{ env_data.CLOUDFRONT_DIST }}
+    ENABLE_STUNNEL_AMAZON_RDS_FIX: True
+    EXAMS_AUDIT_AWS_ACCESS_KEY_ID: __vault__:cache:aws-mitx/creds/read-write-{{ business_unit }}-app-{{ env_data.env_name }}>data>access_key
```

Comment:
Is this encapsulated in the Pulumi managed role now?

Reply:
Will store the current creds statically in vault as this is no longer in use and only a matter of tech debt - https://github.com/mitodl/micromasters/issues/5314